2014/11/06 01:28:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "MainRestController".' in X:\home\localhost\www\rest\framework\web\CWebApplication.php:286
Stack trace:
#0 X:\home\localhost\www\rest\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#1 X:\home\localhost\www\rest\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/rest/
---
2014/11/06 01:30:51 [error] [exception.CHttpException.401] exception 'CHttpException' with message 'Undefined auth user' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:35
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/?api=doclist&q=$clientid
---
2014/11/06 01:38:04 [error] [exception.CHttpException.401] exception 'CHttpException' with message 'Undefined auth user' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:35
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/?api=doclist&q=$clientid
---
2014/11/06 02:02:43 [error] [php] array_key_exists(): The first argument should be either a string or an integer (X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php:188)
Stack trace:
#0 unknown(0): rest\controller\Behavior->renderRest()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(261): call_user_func_array()
#2 X:\home\localhost\www\rest\protected\components\RestController.php(14): MainRestController->__call()
#3 X:\home\localhost\www\rest\protected\components\RestController.php(14): MainRestController->renderRest()
#4 X:\home\localhost\www\rest\protected\controllers\MainRestController.php(21): MainRestController->render()
#5 X:\home\localhost\www\rest\framework\web\actions\CInlineAction.php(49): MainRestController->actionIndex()
#6 X:\home\localhost\www\rest\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 X:\home\localhost\www\rest\framework\web\CController.php(286): MainRestController->runAction()
#8 X:\home\localhost\www\rest\framework\web\CController.php(265): MainRestController->runActionWithFilters()
#9 X:\home\localhost\www\rest\framework\web\CWebApplication.php(282): MainRestController->run()
#10 X:\home\localhost\www\rest\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 X:\home\localhost\www\rest\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 X:\home\localhost\www\rest\index.php(13): CWebApplication->run()
REQUEST_URI=/rest/?api=doclist&q=$clientid
in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php (188)
in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\controller\Behavior.php (73)
in X:\home\localhost\www\rest\protected\components\RestController.php (14)
2014/11/06 02:02:58 [error] [php] array_key_exists(): The first argument should be either a string or an integer (X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php:188)
Stack trace:
#0 unknown(0): rest\controller\Behavior->renderRest()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(261): call_user_func_array()
#2 X:\home\localhost\www\rest\protected\components\RestController.php(14): MainRestController->__call()
#3 X:\home\localhost\www\rest\protected\components\RestController.php(14): MainRestController->renderRest()
#4 X:\home\localhost\www\rest\protected\controllers\MainRestController.php(21): MainRestController->render()
#5 X:\home\localhost\www\rest\framework\web\actions\CInlineAction.php(49): MainRestController->actionIndex()
#6 X:\home\localhost\www\rest\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 X:\home\localhost\www\rest\framework\web\CController.php(286): MainRestController->runAction()
#8 X:\home\localhost\www\rest\framework\web\CController.php(265): MainRestController->runActionWithFilters()
#9 X:\home\localhost\www\rest\framework\web\CWebApplication.php(282): MainRestController->run()
#10 X:\home\localhost\www\rest\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 X:\home\localhost\www\rest\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 X:\home\localhost\www\rest\index.php(13): CWebApplication->run()
REQUEST_URI=/rest/?api=doclist&q=$clientid
in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php (188)
in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\controller\Behavior.php (73)
in X:\home\localhost\www\rest\protected\components\RestController.php (14)
2014/11/06 02:03:05 [error] [php] array_key_exists(): The first argument should be either a string or an integer (X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php:188)
Stack trace:
#0 unknown(0): rest\controller\Behavior->renderRest()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(261): call_user_func_array()
#2 X:\home\localhost\www\rest\protected\components\RestController.php(14): MainRestController->__call()
#3 X:\home\localhost\www\rest\protected\components\RestController.php(14): MainRestController->renderRest()
#4 X:\home\localhost\www\rest\protected\controllers\MainRestController.php(21): MainRestController->render()
#5 X:\home\localhost\www\rest\framework\web\actions\CInlineAction.php(49): MainRestController->actionIndex()
#6 X:\home\localhost\www\rest\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 X:\home\localhost\www\rest\framework\web\CController.php(286): MainRestController->runAction()
#8 X:\home\localhost\www\rest\framework\web\CController.php(265): MainRestController->runActionWithFilters()
#9 X:\home\localhost\www\rest\framework\web\CWebApplication.php(282): MainRestController->run()
#10 X:\home\localhost\www\rest\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 X:\home\localhost\www\rest\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 X:\home\localhost\www\rest\index.php(13): CWebApplication->run()
REQUEST_URI=/rest/?api=doclist&q=$clientid
in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php (188)
in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\controller\Behavior.php (73)
in X:\home\localhost\www\rest\protected\components\RestController.php (14)
2014/11/06 02:03:07 [error] [php] array_key_exists(): The first argument should be either a string or an integer (X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php:188)
Stack trace:
#0 unknown(0): rest\controller\Behavior->renderRest()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(261): call_user_func_array()
#2 X:\home\localhost\www\rest\protected\components\RestController.php(14): MainRestController->__call()
#3 X:\home\localhost\www\rest\protected\components\RestController.php(14): MainRestController->renderRest()
#4 X:\home\localhost\www\rest\protected\controllers\MainRestController.php(21): MainRestController->render()
#5 X:\home\localhost\www\rest\framework\web\actions\CInlineAction.php(49): MainRestController->actionIndex()
#6 X:\home\localhost\www\rest\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 X:\home\localhost\www\rest\framework\web\CController.php(286): MainRestController->runAction()
#8 X:\home\localhost\www\rest\framework\web\CController.php(265): MainRestController->runActionWithFilters()
#9 X:\home\localhost\www\rest\framework\web\CWebApplication.php(282): MainRestController->run()
#10 X:\home\localhost\www\rest\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 X:\home\localhost\www\rest\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 X:\home\localhost\www\rest\index.php(13): CWebApplication->run()
REQUEST_URI=/rest/?api=doclist&q=$clientid
in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php (188)
in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\controller\Behavior.php (73)
in X:\home\localhost\www\rest\protected\components\RestController.php (14)
2014/11/06 02:03:21 [error] [php] Argument 4 passed to RestController::render() must be of the type array, null given, called in X:\home\localhost\www\rest\protected\controllers\MainRestController.php on line 21 and defined (X:\home\localhost\www\rest\protected\components\RestController.php:11)
Stack trace:
#0 X:\home\localhost\www\rest\framework\web\CController.php(308): CInlineAction->runWithParams()
#1 X:\home\localhost\www\rest\framework\web\CController.php(286): MainRestController->runAction()
#2 X:\home\localhost\www\rest\framework\web\CController.php(265): MainRestController->runActionWithFilters()
#3 X:\home\localhost\www\rest\framework\web\CWebApplication.php(282): MainRestController->run()
#4 X:\home\localhost\www\rest\framework\web\CWebApplication.php(141): CWebApplication->runController()
#5 X:\home\localhost\www\rest\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 X:\home\localhost\www\rest\index.php(13): CWebApplication->run()
REQUEST_URI=/rest/?api=doclist&q=$clientid
in X:\home\localhost\www\rest\protected\components\RestController.php (11)
in X:\home\localhost\www\rest\protected\controllers\MainRestController.php (21)
in X:\home\localhost\www\rest\index.php (13)
2014/11/06 02:18:40 [error] [exception.CHttpException.401] exception 'CHttpException' with message 'Undefined auth user' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:35
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/
---
2014/11/06 02:20:26 [error] [exception.CHttpException.401] exception 'CHttpException' with message 'Undefined auth user' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:35
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/
---
2014/11/06 02:20:43 [error] [exception.CHttpException.401] exception 'CHttpException' with message 'Undefined auth user' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:35
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/
---
2014/11/06 02:20:50 [error] [exception.CHttpException.401] exception 'CHttpException' with message 'Undefined auth user' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:35
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/
---
2014/11/06 02:23:13 [error] [exception.CHttpException.401] exception 'CHttpException' with message 'Undefined auth user' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:35
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/
---
2014/11/06 02:23:35 [error] [exception.CHttpException.401] exception 'CHttpException' with message 'Undefined auth user' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:35
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/
---
2014/11/06 02:23:55 [error] [exception.CHttpException.401] exception 'CHttpException' with message 'Undefined auth user' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:35
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/
---
2014/11/06 02:24:07 [error] [exception.CHttpException.401] exception 'CHttpException' with message 'Undefined auth user' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:35
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/
---
2014/11/06 02:24:17 [error] [exception.CHttpException.401] exception 'CHttpException' with message 'Undefined auth user' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:35
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/
---
2014/11/06 02:24:19 [error] [exception.CHttpException.401] exception 'CHttpException' with message 'Undefined auth user' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:35
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/
---
2014/11/06 02:24:47 [error] [exception.CHttpException.401] exception 'CHttpException' with message 'Undefined auth user' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:35
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/
---
2014/11/06 02:24:56 [error] [exception.CHttpException.401] exception 'CHttpException' with message 'Undefined auth user' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:35
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/
---
2014/11/06 02:25:13 [error] [exception.CHttpException.401] exception 'CHttpException' with message 'Undefined auth user' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:35
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/
---
2014/11/06 02:25:42 [error] [exception.CHttpException.401] exception 'CHttpException' with message 'Undefined auth user' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:35
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/
---
2014/11/06 02:26:01 [error] [exception.CHttpException.401] exception 'CHttpException' with message 'Undefined auth user' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:35
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/
---
2014/11/06 02:26:43 [error] [exception.CHttpException.401] exception 'CHttpException' with message 'Undefined auth user' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:35
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/
---
2014/11/06 02:27:09 [error] [exception.CHttpException.401] exception 'CHttpException' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:44
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/
---
2014/11/06 02:27:12 [error] [exception.CHttpException.401] exception 'CHttpException' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:44
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/
---
2014/11/06 02:27:29 [error] [exception.CHttpException.401] exception 'CHttpException' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:44
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/
---
2014/11/06 02:28:42 [error] [exception.CHttpException.401] exception 'CHttpException' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:45
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/
---
2014/11/06 02:33:26 [error] [exception.CHttpException.401] exception 'CHttpException' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:44
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/
---
2014/11/06 02:33:28 [error] [exception.CHttpException.401] exception 'CHttpException' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:44
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/
---
2014/11/06 02:34:34 [error] [exception.CHttpException.401] exception 'CHttpException' with message 'test' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:44
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/
---
2014/11/06 02:34:59 [error] [exception.CHttpException.401] exception 'CHttpException' with message 'Incorrect username or password.' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:44
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/rest/
---
2014/11/06 02:40:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "demo".' in X:\home\localhost\www\rest\framework\web\CWebApplication.php:286
Stack trace:
#0 X:\home\localhost\www\rest\framework\web\CWebApplication.php(141): CWebApplication->runController('demo')
#1 X:\home\localhost\www\rest\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 X:\home\localhost\www\rest\index.php(14): CApplication->run()
#3 {main}
REQUEST_URI=/rest/demo
---
2014/11/06 04:17:41 [error] [exception.CHttpException.401] exception 'CHttpException' with message 'Incorrect username or password.' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:44
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(14): CApplication->run()
#5 {main}
REQUEST_URI=/rest/gii
---
2014/11/06 04:17:42 [error] [exception.CHttpException.401] exception 'CHttpException' with message 'Incorrect username or password.' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:44
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(14): CApplication->run()
#5 {main}
REQUEST_URI=/rest/gii
---
2014/11/06 04:18:11 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection.connectionString cannot be empty.' in X:\home\localhost\www\rest\framework\db\CDbConnection.php:370
Stack trace:
#0 X:\home\localhost\www\rest\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 X:\home\localhost\www\rest\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 X:\home\localhost\www\rest\framework\base\CModule.php(387): CDbConnection->init()
#3 X:\home\localhost\www\rest\framework\base\CModule.php(103): CModule->getComponent('db')
#4 X:\home\localhost\www\rest\framework\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 X:\home\localhost\www\rest\framework\web\CFormModel.php(39): ModelCode->init()
#6 X:\home\localhost\www\rest\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 X:\home\localhost\www\rest\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 X:\home\localhost\www\rest\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 X:\home\localhost\www\rest\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 X:\home\localhost\www\rest\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 X:\home\localhost\www\rest\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 X:\home\localhost\www\rest\framework\web\CWebApplication.php(282): CController->run('index')
#13 X:\home\localhost\www\rest\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 X:\home\localhost\www\rest\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 X:\home\localhost\www\rest\index.php(14): CApplication->run()
#16 {main}
REQUEST_URI=/rest/gii/model/index
HTTP_REFERER=http://localhost/rest/gii/default/index
---
2014/11/06 04:23:04 [error] [exception.CException] exception 'CException' with message 'MainRestController cannot find the requested view "index".' in X:\home\localhost\www\rest\framework\web\CController.php:878
Stack trace:
#0 X:\home\localhost\www\rest\framework\web\CController.php(782): CController->renderPartial('index', Array, true)
#1 X:\home\localhost\www\rest\protected\components\RestController.php(16): CController->render('index', Array, false)
#2 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\controller\Behavior.php(77): RestController->render('index', Array, false)
#3 [internal function]: rest\controller\Behavior->renderRest('index', Array, false, NULL)
#4 X:\home\localhost\www\rest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#5 X:\home\localhost\www\rest\protected\components\RestController.php(14): CComponent->__call('renderRest', Array)
#6 X:\home\localhost\www\rest\protected\components\RestController.php(14): MainRestController->renderRest('index', Array, false, NULL)
#7 X:\home\localhost\www\rest\protected\controllers\MainRestController.php(22): RestController->render('index', Array, false)
#8 X:\home\localhost\www\rest\framework\web\actions\CInlineAction.php(49): MainRestController->actionIndex()
#9 X:\home\localhost\www\rest\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 X:\home\localhost\www\rest\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 X:\home\localhost\www\rest\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 X:\home\localhost\www\rest\framework\web\CWebApplication.php(282): CController->run('')
#13 X:\home\localhost\www\rest\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#14 X:\home\localhost\www\rest\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 X:\home\localhost\www\rest\index.php(14): CApplication->run()
#16 {main}
REQUEST_URI=/rest/
---
2014/11/06 04:32:30 [error] [exception.CHttpException.401] exception 'CHttpException' with message 'Undefined auth user' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:35
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(14): CApplication->run()
#5 {main}
REQUEST_URI=/rest/?api=test&q=1
---
2014/11/06 04:32:41 [error] [exception.CDbException] exception 'CDbException' with message 'Table "users" does not have a column named "username".' in X:\home\localhost\www\rest\framework\db\schema\CDbCommandBuilder.php:694
Stack trace:
#0 X:\home\localhost\www\rest\framework\db\ar\CActiveRecord.php(1518): CDbCommandBuilder->createColumnCriteria(Object(CMysqlTableSchema), Array, '', Array, '`t`.')
#1 X:\home\localhost\www\rest\protected\components\UserIdentity.php(22): CActiveRecord->findByAttributes(Array)
#2 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php(43): UserIdentity->authenticate()
#3 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#4 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#5 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#6 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#7 X:\home\localhost\www\rest\index.php(14): CApplication->run()
#8 {main}
REQUEST_URI=/rest/?api=test&q=1
---
2014/11/06 04:32:57 [error] [php] Trying to get property of non-object (X:\home\localhost\www\rest\protected\controllers\MainRestController.php:19)
Stack trace:
#0 X:\home\localhost\www\rest\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#1 X:\home\localhost\www\rest\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 X:\home\localhost\www\rest\framework\web\CController.php(286): MainRestController->runAction()
#3 X:\home\localhost\www\rest\framework\web\CController.php(265): MainRestController->runActionWithFilters()
#4 X:\home\localhost\www\rest\framework\web\CWebApplication.php(282): MainRestController->run()
#5 X:\home\localhost\www\rest\framework\web\CWebApplication.php(141): CWebApplication->runController()
#6 X:\home\localhost\www\rest\framework\base\CApplication.php(180): CWebApplication->processRequest()
#7 X:\home\localhost\www\rest\index.php(14): CWebApplication->run()
REQUEST_URI=/rest/?api=test&q=1
2014/11/06 04:33:10 [error] [php] Trying to get property of non-object (X:\home\localhost\www\rest\protected\controllers\MainRestController.php:19)
Stack trace:
#0 X:\home\localhost\www\rest\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#1 X:\home\localhost\www\rest\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 X:\home\localhost\www\rest\framework\web\CController.php(286): MainRestController->runAction()
#3 X:\home\localhost\www\rest\framework\web\CController.php(265): MainRestController->runActionWithFilters()
#4 X:\home\localhost\www\rest\framework\web\CWebApplication.php(282): MainRestController->run()
#5 X:\home\localhost\www\rest\framework\web\CWebApplication.php(141): CWebApplication->runController()
#6 X:\home\localhost\www\rest\framework\base\CApplication.php(180): CWebApplication->processRequest()
#7 X:\home\localhost\www\rest\index.php(14): CWebApplication->run()
REQUEST_URI=/rest/?api=test&q=1
in X:\home\localhost\www\rest\protected\controllers\MainRestController.php (19)
in X:\home\localhost\www\rest\index.php (14)
2014/11/06 04:33:36 [error] [exception.CException] exception 'CException' with message 'MainRestController cannot find the requested view "index".' in X:\home\localhost\www\rest\framework\web\CController.php:878
Stack trace:
#0 X:\home\localhost\www\rest\framework\web\CController.php(782): CController->renderPartial('index', Array, true)
#1 X:\home\localhost\www\rest\protected\components\RestController.php(16): CController->render('index', Array, false)
#2 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\controller\Behavior.php(77): RestController->render('index', Array, false)
#3 [internal function]: rest\controller\Behavior->renderRest('index', Array, false, NULL)
#4 X:\home\localhost\www\rest\framework\base\CComponent.php(261): call_user_func_array(Array, Array)
#5 X:\home\localhost\www\rest\protected\components\RestController.php(14): CComponent->__call('renderRest', Array)
#6 X:\home\localhost\www\rest\protected\components\RestController.php(14): MainRestController->renderRest('index', Array, false, NULL)
#7 X:\home\localhost\www\rest\protected\controllers\MainRestController.php(26): RestController->render('index', Array, false)
#8 [internal function]: MainRestController->actionIndex('test', '1')
#9 X:\home\localhost\www\rest\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(MainRestController), Array)
#10 X:\home\localhost\www\rest\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(MainRestController), Object(ReflectionMethod), Array)
#11 X:\home\localhost\www\rest\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 X:\home\localhost\www\rest\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 X:\home\localhost\www\rest\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 X:\home\localhost\www\rest\framework\web\CWebApplication.php(282): CController->run('')
#15 X:\home\localhost\www\rest\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#16 X:\home\localhost\www\rest\framework\base\CApplication.php(180): CWebApplication->processRequest()
#17 X:\home\localhost\www\rest\index.php(14): CApplication->run()
#18 {main}
REQUEST_URI=/rest/?api=test&q=1
---
2014/11/06 04:33:52 [error] [exception.CDbException] exception 'CDbException' with message 'Table "users" does not have a column named "username".' in X:\home\localhost\www\rest\framework\db\schema\CDbCommandBuilder.php:694
Stack trace:
#0 X:\home\localhost\www\rest\framework\db\ar\CActiveRecord.php(1518): CDbCommandBuilder->createColumnCriteria(Object(CMysqlTableSchema), Array, '', Array, '`t`.')
#1 X:\home\localhost\www\rest\protected\components\UserIdentity.php(22): CActiveRecord->findByAttributes(Array)
#2 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php(43): UserIdentity->authenticate()
#3 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#4 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#5 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#6 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#7 X:\home\localhost\www\rest\index.php(14): CApplication->run()
#8 {main}
REQUEST_URI=/rest/?api=test&q=1
---
2014/11/06 04:35:31 [error] [exception.CHttpException.401] exception 'CHttpException' with message 'Incorrect username or password.' in X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\service\auth\adapters\Basic.php:44
Stack trace:
#0 X:\home\localhost\www\rest\protected\extensions\yii-rest-api\library\rest\Service.php(84): rest\service\auth\adapters\Basic->authenticate()
#1 X:\home\localhost\www\rest\framework\base\CComponent.php(561): rest\Service->onBeginRequest(Object(CEvent))
#2 X:\home\localhost\www\rest\framework\base\CApplication.php(207): CComponent->raiseEvent('onBeginRequest', Object(CEvent))
#3 X:\home\localhost\www\rest\framework\base\CApplication.php(178): CApplication->onBeginRequest(Object(CEvent))
#4 X:\home\localhost\www\rest\index.php(14): CApplication->run()
#5 {main}
REQUEST_URI=/rest/?api=doclist&q=1
---
2014/11/06 04:50:55 [error] [exception.CHttpException.405] exception 'CHttpException' with message 'Parameter 'q' can not be empty' in X:\home\localhost\www\rest\protected\controllers\MainRestController.php:14
Stack trace:
#0 X:\home\localhost\www\rest\framework\web\actions\CInlineAction.php(49): MainRestController->actionIndex()
#1 X:\home\localhost\www\rest\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#2 X:\home\localhost\www\rest\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#3 X:\home\localhost\www\rest\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#4 X:\home\localhost\www\rest\framework\web\CWebApplication.php(282): CController->run('')
#5 X:\home\localhost\www\rest\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#6 X:\home\localhost\www\rest\framework\base\CApplication.php(180): CWebApplication->processRequest()
#7 X:\home\localhost\www\rest\index.php(14): CApplication->run()
#8 {main}
REQUEST_URI=/rest/?api=doclist
---
